metricbeat/module/kibana/stats/_meta/data.json

```diff
@@ -1,73 +1,122 @@
     "kibana": {
+        "elasticsearch": {
+            "cluster": {
+                "id": "zJpGlM76SkSnRMK1Ihihnw"
+            }
+        },
         "stats": {
-            "concurrent_connections": 1,
+            "concurrent_connections": 0,
             "host": {
-                "name": "localhost"
+                "name": "0"
+            },
+            "index": "kibana",
+            "kibana": {
+                "host": "0",
+                "index": ".kibana",
+                "name": "kibana",
+                "snapshot": false,
+                "status": "green",
+                "transport_address": "0:5601",
+                "uuid": "47c5b96a-e3c4-430c-afee-5c7e27683b80",
+                "version": "7.9.0"
+            },
+            "name": "kibana",
+            "os": {
+                "distro": "Centos",
+                "distroRelease": "Centos-7.8.2003",
+                "load": {
+                    "15m": 1.1416015625,
+                    "1m": 1.18994140625,
+                    "5m": 1.28466796875
+                },
+                "memory": {
+                    "free_in_bytes": 11949064192,
+                    "total_in_bytes": 33300463616,
+                    "used_in_bytes": 21351399424
+                },
+                "platform": "linux",
+                "platformRelease": "linux-5.6.10-arch1-1",
+                "uptime_in_millis": 11437000
             },
-            "index": "Shaunaks-MBP-2.attlocal.net",
-            "name": "Shaunaks-MBP-2.attlocal.net",
             "process": {
                 "event_loop_delay": {
-                    "ms": 0.45250000059604645
+                    "ms": 0.13968499936163425
                 },
                 "memory": {
                     "heap": {
                         "size_limit": {
                             "bytes": 1526909922
                         },
                         "total": {
-                            "bytes": 307593216
+                            "bytes": 475709440
                         },
                         "used": {
-                            "bytes": 226887112
+                            "bytes": 337236176
                         }
+                    },
+                    "resident_set_size": {
+                        "bytes": 592297984
                     }
                 },
                 "uptime": {
-                    "ms": 124098
+                    "ms": 373166
                 }
             },
             "request": {
                 "disconnects": 0,
-                "total": 0
+                "total": 4
             },
             "response_time": {
-                "avg": {},
+                "avg": {
+                    "ms": 5
+                },
                 "max": {
-                    "ms": 0
+                    "ms": 6
                 }
             },
             "snapshot": false,
-            "status": "green"
+            "status": "green",
+            "usage": {
```

Comment:
With the recent changes in https://github.com/elastic/beats/pull/22732, I don't think we even need to collect `usage` any more. WDYT @chrisronline - safe to omit `usage` collection in this PR here?

Reply:
Yes! Please remove it